x/authority/keeper/chain_info.go

```diff
@@ -0,0 +1,26 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/zetacore/x/authority/types"
+)
+
+// SetChainInfo sets the chain info to the store
+func (k Keeper) SetChainInfo(ctx sdk.Context, chainInfo types.ChainInfo) {
```

Comment:
How would this be set?

Reply:
https://github.com/zeta-chain/node/issues/2271